webpack.config.js

```diff
@@ -4,12 +4,12 @@ const HtmlWebPackPlugin = require("html-webpack-plugin");
 module.exports = {
     mode: "development",
     entry: {
-        app: "./src/index.ts",
+        app: ["./src/index.ts", "./src/js/load-pyodide.js", "./src/js/pyodide-controller.js"],
```

Comment:
Always load js and ts files here from now on! If you add a separate entry for each file it will reload the entire bundle multiple times which will cause a variety of errors (mostly various parts of code running multiple times).